src/base/utils/gzip.h

```diff
@@ -36,8 +36,8 @@ namespace Utils
 {
     namespace Gzip
     {
-        bool compress(QByteArray src, QByteArray &dest);
-        bool uncompress(QByteArray src, QByteArray &dest);
+        QByteArray compress(const QByteArray &data, bool *ok = nullptr, int level = 6);
```

Comment:
Did not find where you call it with `ok == nullptr`. Why it is optional then? Maybe make it a reference or return a pair of values?